src/memory.rs

```diff
@@ -98,6 +98,19 @@ impl Pointer {
     pub fn never_ptr() -> Self {
         Pointer::new(NEVER_ALLOC_ID, 0)
     }
+    
+    pub fn is_null_ptr(&self) -> bool {
+        // FIXME: Is this the right way?
+        return *self == Pointer::from_int(0)
```

Comment:
Yes, that's correct. It's basically what happens when miri interprets the Mir of is_null